lib/fortnox/api/invoice.rb

```diff
@@ -0,0 +1,31 @@
+require "fortnox/api/models/invoice"
+require "fortnox/api/repositories/invoice"
+require "fortnox/api/validators/invoice"
+
+module Fortnox
+  module API
+    class Invoice < Fortnox::API::Model::Invoice
+
+      def valid?
+        Fortnox::API::Validator::Invoice.validate( self )
```

Comment:
Sorry, I was in EDIInformation land :/ Ofc the Invoice model should be validated properly ...